### ISSUE TITLE
Team stats range option

### ISSUE THREAD

**Issue by barrelman70** [jldbc/pybaseball#345]
Any way to have something like team_pitching_range and team_batting_range like there is for pitching_stats_range.  I need to generate game-by-game aggregate historical team stats to feed some machine learning.  Or do you know of an historical db somewhere I can grab?

**Comment by erin2722** [comment]
I don't think that baseball reference (which is the current source of range data) has team aggregate data for daily games, but if #346 is merged, that should address this issue.
